2.4.41testenv/test/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# 
+# Test the OpenLDAP test-env image.
+#
+# IMAGE_NAME specifies the name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
```

Comment:
This shouldn't be empty - d'oh. This is the test image, layered on the OpenLDAP image and with a pre-imported set of data that will be used for testing.